manifests/router/link_route_pattern.pp

```diff
@@ -6,10 +6,13 @@
 #
 # $prefix::     Prefix to use
 #
+# $dir::        Direction when using asymmetric routing
```

Comment:
Could this be 'direction' since 'dir' means 'directory' in so many other languages and libraries?